src/components/TheBankAccountList.vue

```diff
@@ -17,10 +17,7 @@
           <fa-icon icon="sync" />
         </span>
       </a>
-      <div
-        class="notification is-danger is-light"
-        v-if="hasAccountsLoadingError"
-      >
+      <div class="notification is-danger is-light" v-if="accountsLoadingError">
```

Comment:
Are you sure this should be kept on one line ? Did you Eslint your changes (You did format code in the first change listed in the commit).

Reply:
You are right, Eslint automatically reformatted the line. i think Eslint formatted it that way because `accountsLoadingError` is shorter than `hasAccountsLoadingError` and all this hunk can feet in one line. I'll be more vigilant for the next commits. 
Thank you for the review !

Reply:
Oh you mean that your code editor lints your code on save ?
If that's the case then you can discard my comments.

Anyway, thanks for the refactoring. Nice.